3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,159 @@
+
+//Item array removal
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  
+  const indexToRemove = names.indexOf(nameToRemove);
+
+  if (indexToRemove !== -1) {
+    names.splice(indexToRemove, 1)
+  }
+  
+  console.log(names); 
+
+
+//When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+
+  function calculateTime (travelInformation){
+    const { speed, destinationDistance } = travelInformation;
+    
+    const totalTimeInHours = destinationDistance / speed;
+
+    const hours = Math.floor(totalTimeInHours);
+    const minutes = Math.round((totalTimeInHours - hours) * 60);
+
+    const formattedTime = `${hours} hours and ${minutes} minutes`;
+
+    return formattedTime;
+
+  }
+  
+  const travelTime = calculateTime(travelInformation);
+  console.log(travelTime); 
+
```

Comment:
✅ Fantastic job! This solution demonstrates a clear understanding of the problem and a neat approach to solving it. You've structured your code effectively and made excellent use of ES6 destructuring.
❌ **Function Naming:** The function name `calculateTime` is concise and does convey its purpose. However, consider being a bit more descriptive, like `calculateTravelTime`, as it provides more context about what time you're calculating.
☠️ **Indentation**: Ensure consistent indentation throughout your code. Proper indentation enhances readability.